UM/Settings/ContainerRegistry.py

```diff
@@ -4,6 +4,7 @@
 import os
 import re #For finding containers with asterisks in the constraints.
 import urllib
+import pickle
```

Comment:
I really like Ghostkeepers style of adding a comment why we use certain libs. In the case of pickle, that might be a good one (as I never heard of it before seeing it here)